src/main/java/io/javalin/core/JavalinConfig.java

```diff
@@ -66,6 +66,7 @@
         @Nullable public WsHandler wsLogger = null;
         @Nullable public Server server = null;
         @Nullable public Consumer<ServletContextHandler> servletContextHandlerConsumer = null;
+        @Nullable public DynamicCompressionStrategy dynamicCompressionStrategy = null;
```

Comment:
Must default to null for backwards compatibility. 

This is because the DynamicCompressionHandler object will respect the old dynamicGzip flag ONLY IF dynamicCompressionStrategy is null